packages/cli/src/commands/generate/scaffold/templates/pages/NewNamePage.js.template

```diff
@@ -1,12 +1,7 @@
-import ${pluralPascalName}Layout from 'src/layouts/${pascalScaffoldPath}${pluralPascalName}Layout'
 import New${singularPascalName} from 'src/components/${pascalScaffoldPath}New${singularPascalName}'
 
 const New${singularPascalName}Page = () => {
-  return (
-    <${pluralPascalName}Layout>
-      <New${singularPascalName} />
-    </${pluralPascalName}Layout>
-  )
+  return <New${singularPascalName} />
```

Comment:
@cannikin do you think these pages are starting to look a little too empty? This would be for another PR, but should we just move `NewPost` into `NewPostPage` now that Sets take the Layout out?

![image](https://user-images.githubusercontent.com/32992335/119236039-1a392b80-baea-11eb-988a-0d6b3d5e13a2.png)

Reply:
After taking another look at this, this is the only page we could do this for (all the others are Cells). I feel like we might as well wait.